tests/ext-activitylog.test.js

```diff
@@ -269,3 +270,77 @@ test('clearing logs from activitylog page', async () => {
   expect(activityLog.model.logs).toMatchObject([]);
   expect(getTableRows().length).toBe(0);
 });
+
+test('timestamp is formatted and rendered correctly', () => {
+  const logTimestamp = 1597686226302;
+
+  const logs = [
+    {
+      /* renders 1st row in table */
+      id: 'id1@test',
+      viewType: 'viewType@test',
+      type: 'type@test',
+      data: [{ test: 'test1@data' }],
+      timeStamp: logTimestamp,
+    },
+  ];
+
+  const addListener = jest.fn();
+  const sendMessage = jest.fn();
+  const connect = jest.fn();
+
+  window.browser = {
+    runtime: {
+      onMessage: { addListener },
+      sendMessage,
+      connect,
+    },
+  };
+
+  sendMessage.mockImplementation(() => {
+    return Promise.resolve({ existingLogs: [] });
+  });
+
+  document.body.innerHTML = activityLogBody;
+  const dateTimeFormatFn = jest.spyOn(ExtListen, 'dateTimeFormat');
```

Comment:
You shouldn't mock your function, but the API that you're using. The only changes should be:
- fix `'en-US'` as the locale instead of `undefined`.
- fix the `timeZone` option.

By doing both, the generated time strings become fully deterministic / predictable, and you can put the expected times in strings below.